### ISSUE TITLE
unable to load project in sts

### ISSUE THREAD

**Issue by dhirajthakur13027** [serphacker/serposcope#229]
hello everyone,

i imported  serposcope in sts...its imported but im unable to make jar file of it or run it on sts. its shows error in  sts in particulary in core. as in core  we are trying to import querybuilder package but in src/main/java but there is querybuilder package @noguespi @ron51.....kindly help.

![1](https://user-images.githubusercontent.com/64984253/83778441-5ccee180-a6a8-11ea-8f18-1960ec51e630.png)
![2](https://user-images.githubusercontent.com/64984253/83778448-5e000e80-a6a8-11ea-9ba8-f31e4e9598c4.png)
![3](https://user-images.githubusercontent.com/64984253/83778451-5e98a500-a6a8-11ea-9b95-2f6dbc10deed.png)
![4](https://user-images.githubusercontent.com/64984253/83779833-0f537400-a6aa-11ea-9915-aa70a4822fe1.png)

**Comment by jinjie** [comment]
Hi.. I do not use STS. I am quite new to building in Java too. I successfully build the project by following closely on the instructions at https://github.com/serphacker/serposcope/blob/master/BUILD-INSTRUCTIONS.md

**Comment by ron51** [comment]
@noguespi Can you check this out, we have a developer who can contribute to this project. If you can help then we will be able to resolve issues quickly and maintain this project.

**Comment by jinjie** [comment]
@ron51 I have tried building from the `master` branch. It seem like there is 2 issues but not the error from your screenshot.

Can you try not using STS to narrow down the problems? I suggest running from the command line: `mvn -Dcodegen clean install -U` in the project root.

There is a package `javax.annotations` dependency missing, so I added the following in `core/pom.xml`.

```
        <dependency>
            <groupId>javax.annotation</groupId>
            <artifactId>javax.annotation-api</artifactId>
            <version>1.2</version>
        </dependency>
```

Do let me know how things works out.

**Comment by noguespi** [comment]
also it requires java 8

**Comment by dhirajthakur13027** [comment]
yes its resolved.i followed all build instructions..now i could deploy it locally..but i couldnot do it in superdev mode..there was an error like no goals defined when i ran following command in serposcope/web folder

mvn ninja:run \
-Dninja.mainClass=serposcope.lifecycle.Daemon \
-Dninja.jvmArgs="-Dlogback.configurationFile=dev/logback-stdout.xml -Dserposcope.datadir=/var/tmp/serposcope -Dserposcope.db.options=;AUTO_SERVER=TRUE"


so i did only "mvn ninja:run"..
im able to see UI of it

**Comment by ron51** [comment]
@noguespi Can we schedule a call on Zoom? If yes, then please forward me your email id, I will send you an email with timings and invite.
